libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -203,11 +214,9 @@ private List<String> toExitCode(ContraIndicators contraIndicators)
                 .toList();
     }
 
-    private List<String> toAlwaysRequiredExitCode(ContraIndicators contraIndicators)
-            throws UnrecognisedCiException {
-        return toExitCode(contraIndicators).stream()
-                .filter(configService.getSsmParameter(ALWAYS_REQUIRED_EXIT_CODES)::contains)
-                .toList();
+    private boolean breachingCiThreshold(ContraIndicators contraIndicators) {
```

Comment:
It feels a bit weird for this method to be here (surely we check breaching CI threshold elsewhere too), but perhaps it's not easy to share - and it's simple enough not to be a big deal.

Reply:
It does feel like something we should "know" at this point from the place where we do this check - ci-scoring. But we don't store the result in the users session or anything. Perhaps we could - but then i have fears about that leading to hard to detect bugs later down the line when relying on it.

Reply:
Yeah - I don't think we should try and cache it (since it's such a simple calculation), I was more thinking about sharing the code, so we have one method for checking CI score breaches.

It's one integer comparison though, so not that fussed about it :)